tests/Microsoft.Identity.Test.Unit.net45/ApiConfigTests/Harnesses/AbstractBuilderHarness.cs

```diff
@@ -51,7 +52,9 @@ internal abstract class AbstractBuilderHarness
             CoreAssert.AreScopesEqual(
                 (expectedScopes ?? MsalTestConstants.Scope).AsSingleString(),
                 CommonParametersReceived.Scopes.AsSingleString());
-            CollectionAssert.AreEqual(expectedExtraQueryParameters, CommonParametersReceived.ExtraQueryParameters);
+            CollectionAssert.AreEqual(
+                expectedExtraQueryParameters, 
+                CommonParametersReceived.ExtraQueryParameters.ToDictionary(kvp => kvp.Key, kvp => kvp.Value));
```

Comment:
ExtraQueryParameters is already a dictionary.  Why the ToDictionary() call here?

Reply:
The old "ICollection" interface is not implemented by IDictionary. IDictionary implemenbts ICollection<KeyValuePair<K,V>> and not ICollection<T>. 

Still, I can simplify this by calling a ToList() on the dictionary.